### PR TITLE
[Merged by Bors] - feat: port CategoryTheory.Conj

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by mo271** [comment]
The only thing that is left is a non-working rfl in `conjAut_apply`

File: Mathlib/CategoryTheory/Conj.lean [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+/-- `conj` defines a group isomorphisms between groups of automorphisms -/
+def conjAut : Aut X ≃* Aut Y :=
+  (Aut.unitsEndEquivAut X).symm.trans <| (Units.mapEquiv α.conj).trans <| Aut.unitsEndEquivAut Y
+set_option linter.uppercaseLean3 false in
+#align category_theory.iso.conj_Aut CategoryTheory.Iso.conjAut
+
+theorem conjAut_apply (f : Aut X) : α.conjAut f = α.symm ≪≫ f ≪≫ α := by
+  cases f; cases α; apply CategoryTheory.Iso.ext 
+  simp only [conjAut, Aut.unitsEndEquivAut, conj]; rfl
```
Comment (mattrobball) [inline Mathlib/CategoryTheory/Conj.lean:130]: Lean could not see through these definitions.

**Comment by mattrobball** [comment]
Should be ready to go.

File: Mathlib/CategoryTheory/Conj.lean (hunk 1 above)
Comment (mo271) [reply Mathlib/CategoryTheory/Conj.lean:130]: great!

**Review by urkud — COMMENTED** [review]
Thanks! I have a few minor comments, otherwise LGTM.

File: Mathlib/CategoryTheory/Conj.lean (hunk 1 above)
Comment (mo271) [reply Mathlib/CategoryTheory/Conj.lean]: yes, `aesop_cat` works!

File: Mathlib/CategoryTheory/Conj.lean [hunk 2]
```diff
@@ -32,34 +32,35 @@ namespace Iso
 variable {C : Type u} [Category.{v} C]
 
 /-- If `X` is isomorphic to `X₁` and `Y` is isomorphic to `Y₁`, then
-there is a natural bijection between `X ⟶ Y` and `X₁ ⟶ Y₁`. See also `equiv.arrow_congr`. -/
+there is a natural bijection between `X ⟶ Y` and `X₁ ⟶ Y₁`. See also `Equiv.arrowCongr`. -/
 def homCongr {X Y X₁ Y₁ : C} (α : X ≅ X₁) (β : Y ≅ Y₁) : (X ⟶ Y) ≃ (X₁ ⟶ Y₁)
     where
```
[the remark this answers is not in the document]
Comment (mo271) [reply Mathlib/CategoryTheory/Conj.lean]: done

File: Mathlib/CategoryTheory/Conj.lean [hunk 3]
```diff
@@ -32,34 +32,35 @@ namespace Iso
...
+      rw [Category.assoc, Category.assoc, β.hom_inv_id, α.hom_inv_id_assoc, Category.comp_id]
   right_inv f :=
-    show α.inv ≫ (α.Hom ≫ f ≫ β.inv) ≫ β.Hom = f by
-      rw [category.assoc, category.assoc, β.inv_hom_id, α.inv_hom_id_assoc, category.comp_id]
+    show α.inv ≫ (α.hom ≫ f ≫ β.inv) ≫ β.hom = f by
+      rw [Category.assoc, Category.assoc, β.inv_hom_id, α.inv_hom_id_assoc, Category.comp_id]
 #align category_theory.iso.hom_congr CategoryTheory.Iso.homCongr
 
+-- @[simp, nolint simpNF] -- Porting note: dsimp can not prove this
 @[simp]
```
[the remark this answers is not in the document]
Comment (mo271) [reply Mathlib/CategoryTheory/Conj.lean:51]: done

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**